UseCases/Sources/ManualContextPropagation/main.swift

```diff
@@ -2,9 +2,10 @@ import ContextPropagation
 
 // MARK: - Demo
 
-let server = FakeHTTPServer(
-    instrumentationMiddlewares: [FakeTracer.Middleware(tracer: FakeTracer())]
-) { context, request, client -> FakeHTTPResponse in
+let server = FakeHTTPServer(instrumentationMiddleware: MultiplexInstrumentationMiddleware([
+    InstrumentationMiddleware(FakeTracer.Middleware(tracer: FakeTracer())),
+    InstrumentationMiddleware(SecondFakeTracer.Middleware(tracer: SecondFakeTracer()))
+])) { context, request, client -> FakeHTTPResponse in
```

Comment:
maybe no need to show the first/second stuff in the samples? We have the test for that, we know it works :)